### PR TITLE
Added Client Telemetry Usage Information

### PR DESCRIPTION
**Pull request by kushagraThapar**
Added Client Telemetry Usage Information

### CODE REVIEW

**Review by kirankumarkolli — APPROVED** [review]

File: sdk/cosmos/azure-cosmos-benchmark/src/test/java/com/azure/cosmos/benchmark/linkedin/data/ClientTelemetryTest.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+     *
+     * Usage of Client Telemetry Endpoint in various environments:
+     *   Production Endpoint: https://tools.cosmos.azure.com/api/clienttelemetry/trace _(recommended)_
+     *   Stage Endpoint: https://tools-staging.cosmos.azure.com/api/clienttelemetry/trace/ _(recommended for local Testing)_
+     *   Test Endpoint: https://juno-test.documents-dev.windows-int.net/api/clienttelemetry/trace/ _(Allows only dogfood accounts i.e. https://df.onecloud.azure-test.net/)_
+     */
+    private void enableClientTelemetry() {
+        String clientTelemetryProductionEndpoint = "https://tools.cosmos.azure.com/api/clienttelemetry/trace/";
+        String clientTelemetryStagingEndpoint = "https://tools-staging.cosmos.azure.com/api/clienttelemetry/trace/";
+        String clientTelemetryTestEndpoint = "https://juno-test.documents-dev.windows-int.net/api/clienttelemetry/trace/";
```
Comment (xinlian12) [inline sdk/cosmos/azure-cosmos-benchmark/src/test/java/com/azure/cosmos/benchmark/linkedin/data/ClientTelemetryTest.java:32]: I thought they have 3 test endpoints. Juno-test, juno-test2, juno-test3

File: sdk/cosmos/azure-cosmos-benchmark/src/test/java/com/azure/cosmos/benchmark/linkedin/data/ClientTelemetryTest.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        CosmosClient cosmosClient = new CosmosClientBuilder()
+            .endpoint(TestConfigurations.HOST)
+            .key(TestConfigurations.MASTER_KEY)
+            .directMode()
+            .clientTelemetryEnabled(true)
+            .buildClient();
+    }
+
+    /**
+     * Disallow Query Plan Generation for single partition queries
```
Comment (xinlian12) [inline sdk/cosmos/azure-cosmos-benchmark/src/test/java/com/azure/cosmos/benchmark/linkedin/data/ClientTelemetryTest.java:44]: will this be allowed for all types query? like orderby, aggregates etc

**Comment by kushagraThapar** [comment]
Oh really, didn't know this. 
Also, I see I enabled auto merge on my PR - which merged it automatically.. sorry if the PR was merged too soon.. Please feel free to add more comments, and I can put a follow up PR on this to iterate over your comments.

**Comment by xinlian12** [comment]
> Oh really, didn't know this.
> Also, I see I enabled auto merge on my PR - which merged it automatically.. sorry if the PR was merged too soon.. Please feel free to add more comments, and I can put a follow up PR on this to iterate over your comments.

No worry,  yea, that is what I remembered. But I do not really think that matters as for external customers, they can not really have a test account

**Pull request merged**